### ISSUE TITLE
Windows Support radio option doesn't become active if I open Cluster Option from different RKE provider

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#16397]
Version - 2.2. master 10/31

**Steps to reproduce (least amount of steps as possible):**
1.  Go to Add Cluster
2.  Click on Amazon EC2 and open Cluster Option accordion
3.  Click on Custom
4.  Change Network Provider to Flannel

**Result:**  the Windows Support radio buttons do not become available.
![image](https://user-images.githubusercontent.com/11514927/47821513-2c7dcf00-dd1e-11e8-8119-0deeca90c3c0.png)

**Comment by tfiduccia** [comment]
Version - 2.2 master 11/14
This is still happening.
Steps:
1.  Add cluster
2.  Select Amazon EC2
3.  Open Cluster Options accordion
4.  Change cluster option to Custom cluster
5.  Change Network Provider to Flannel

![image](https://user-images.githubusercontent.com/11514927/48504648-9019f900-e802-11e8-8205-d58b786840c3.png)

**Comment by tfiduccia** [comment]
Still broken on 2.2 master 11/27.  Same steps as above
![image](https://user-images.githubusercontent.com/11514927/49119271-e0fb0a00-f264-11e8-9d92-f2117869a1e9.png)

**Comment by tfiduccia** [comment]
Light theme issue

**Comment by davidnuzik** [comment]
Version: master 2.2 (1/4/19)
Confirmed fix. I no longer see Windows Support radio buttons grayed-out (disabled) when choosing a custom rke cluster and Flannel. I confirmed that upon creation we still have the option to pick Linux/Windows under Node Operating System and get an appropriate PowerShell command. When I choose flannel but choose to keep Windows Support Disabled I do not have an option for Windows as expected.
